packages/devtools_app/lib/src/screens/debugger/debugger_controller.dart

```diff
@@ -230,9 +230,9 @@ class DebuggerController extends DisposableController
     _scriptLocation.value = scriptLocation;
   }
 
-  Future<Script> getScriptForRef(ScriptRef ref) async {
-    final cachedScript = scriptManager.getScriptCached(ref);
-    if (cachedScript == null && ref != null) {
+  Future<Script?> getScriptForRef(ScriptRef? ref) async {
+    final cachedScript = scriptManager.getScriptCached(ref!);
```

Comment:
if we are unconditionally null asserting `ref` here, then let's make the parameter be non-nullable